aries-site/src/examples/components/namevaluelist/NameValueListAnatomy.js

```diff
@@ -45,7 +45,8 @@ const connections = [
 
 const AnatomyGrid = ({ ...rest }) => (
   <Grid
-    columns={['xxsmall', ['small', 'medium']]}
+    columns={['medium']}
```

Comment:
we can't change these here because it's being used in NameValueList

Reply:
yes that change shouldnt be in there my mistake